pkg/webui/console/views/device-add/manual/form/form.js

```diff
@@ -246,10 +294,57 @@ const ManualForm = props => {
   }, [])
 
   const [freqPlan, setFreqPlan] = React.useState()
+  const freqPlanRef = React.useRef(freqPlan)
   const handleFreqPlanChange = React.useCallback(band => {
     setFreqPlan(band.value)
   }, [])
 
+  const [phyVersion, setPhyVersion] = React.useState()
+  const phyVersionRef = React.useRef(phyVersion)
+  const handlePhyVersionChange = React.useCallback(version => {
+    setPhyVersion(version)
+  }, [])
+
+  React.useEffect(() => {
+    const updateMacSettings = async () => {
+      try {
+        const settings = await getDefaultMacSettings(freqPlan, phyVersion)
+        setMacSettings(settings)
+      } catch (err) {}
```

Comment:
This needs to be handled. Let's use at least a toast notification stating `Unable to load default mac settings for the "${freqPlan}" frequency plan.` And we probably also want to reset the mac settings in that case.